priv/static/index.html

```diff
@@ -30,51 +30,66 @@
     <nav>
       <ul>
         <li class="navbar"><a href="/index.html">tournamap.gg</a></li>
-        <li class="navbar" id="about-btn">About</li>
+        <li class="navbar" id="about-btn"><a href="#">About</a></li>
```

Comment:
What's the point of this a tag? If it's to make the cursor appear clickable, what's the advantage of doing it this way as opposed to what I did in 7688e3359bc8d3632542bda87affe63b304bf39c? (I'm not against doing it this way instead, I just don't understand it yet.)

Reply:
When I was testing it, I had different cursor effects on the tournamap.gg <li> and the about <li>, I could be mistaken with my test. At the time I had added it to maintain consistency with the appearance of the cursor.

Reply:
Try syncing your fork if you haven't, I think your original branch may have been done before I fixed it via CSS. If it's still presenting an issue, can you take a screenshot? We might just be talking about different things.